sp800-63-3/cover.md

```diff
@@ -91,14 +91,14 @@ http://dx.doi.org/10.6028/NIST.SP.XXX
 
 {::comment}
 
-**Comments on this publication may be submitted to eauth-comments@nist.gov  
+**Comments on this publication may be submitted to eauth-comment@nist.gov  
```

Comment:
@kboeckl, can you get a new email address? I don't know if this one is an alias or a full inbox. whatever address paul fancies. thanks!